bono.yaml

```diff
@@ -137,6 +179,15 @@ resources:
 
             # Log all output to file.
             exec > >(tee -a /var/log/clearwater-heat-bono.log) 2>&1
+            set -x
+
+            # Set up the signaling network namespace
+            ip netns add signaling
+            ip link set eth1 netns signaling
```

Comment:
Is `eth1` guaranteed to be the signaling network? Is this related to the ordering of the ports in the `networks` section for this server?